agent/session.go

```diff
@@ -205,22 +206,68 @@ func (s *session) handleSessionMessage(ctx context.Context, msg *api.SessionMess
 }
 
 func (s *session) watch(ctx context.Context) error {
-	log.G(ctx).Debugf("(*session).watch")
-	client := api.NewDispatcherClient(s.conn)
-	watch, err := client.Tasks(ctx, &api.TasksRequest{
-		SessionID: s.sessionID})
-	if err != nil {
-		return err
-	}
+	log := log.G(ctx).WithFields(logrus.Fields{"method": "(*session).watch"})
+	log.Debugf("")
```

Comment:
We wanted to log the call to `(*session).watch` (see https://github.com/docker/swarmkit/pull/1508/files#diff-15ffe95a2da45f70696ffa3c01949601L208), just to indicate that it was starting, but we also wanted those fields to be attached to the logger so it can be used to log errors later.  This call to `Debugf` just logs the fields of the logger without any additional info.